mantidproject/mantid
# Fix some sphinx warnings

Try to fix the sphinx warnings

**To test:**

All doctests should pass. If you build the documentation, you should see no warnings

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
